### ISSUE TITLE
Cooking::Cook_Book dataset incorrectly using STORE_PRODUCT lookup

### ISSUE THREAD

**Issue by ghost** [rojadyn/portia-miner#8]
Take the following (broken) recipes:

                Recipe: Bubblefish Stew
           Ingredients: 1: Tea x1
                        2: Emperor Bubblefish x2
                        3: Milk x1
      **Quality>Cost**: 35
    **Quality>Weight**: 10
            **Weight**: 212

                Recipe: Bubblefish Stew
           Ingredients: 1: Tea x1
                        2: Bubblefish x2
                        3: Milk x1
      **Quality>Cost**: 35
    **Quality>Weight**: 10
            **Weight**: 213

Where'd the "Milk x1" come from? Well, take a look at the DB rows from `Cook_Book`:

    ID	Weight	Quality	Food	Material
    26	212	1	2000038	4000088_1;4000249_2;101_1
    27	213	1	2000038	4000088_1;4000068_2;101_1

The `Cook_Book.py` dataset is trying to resolve "101_1" and it's getting a hit on `Lookup.STORE_PRODUCT` (which happens to be Milk).

Why is that Lookup in the list of lookups for resolving `Material`? Based on the disassembly of game code's `CookBookMgr`, if the MaterialID is greater than 999999 then it assumes it is an item/prop ID, else it assumes it is a Tag from `Cook_TagList` and searches for a matching ID in there.

TL;DR: Not sure why `Lookup.STORE_PRODUCT` is in the list of lookups for `Cook_Book.py` - seems like that is not desired and should be removed.

**Comment by ghost** [comment]
Forgot to mention... I made the suggested change locally and spot-checked the same two recipes in Cook_book.txt - they now appear as expected:

                Recipe: Bubblefish Stew
           Ingredients: 1: Tea x1
                        2: Emperor Bubblefish x2
                        3: Random Seasoning x1
      **Quality>Cost**: 35
    **Quality>Weight**: 10
            **Weight**: 212

                Recipe: Bubblefish Stew
           Ingredients: 1: Tea x1
                        2: Bubblefish x2
                        3: Random Seasoning x1
      **Quality>Cost**: 35
    **Quality>Weight**: 10
            **Weight**: 213

Since I'm not sure what recipe(s) would have driven `Lookup.STORE_PRODUCT` to be added, I'm not sure what else to check in order to verify removing it didn't break something else...
